.github/ISSUE_TEMPLATE/new_bug.md

```diff
@@ -0,0 +1,32 @@
+---
+name: New Bug
+about: Identification of a new issue
+title: 'New Bug'
+labels: 
+assignees: '@abbylindsay'
+
+---
```

Comment:
We usually want to track the version information of the bug was identified in. For example, app version 1.2 if know. Or firmware 4.0.1.